packages/roosterjs-editor-plugins/test/TableResize/tableData.ts

```diff
@@ -0,0 +1,8 @@
+export const DEFAULT_TABLE_MERGED =
+    '<table cellspacing="0" cellpadding="1" style="border-collapse: collapse; box-sizing: border-box; width: 493px; height: 105.5px;"><tbody><tr><td style="width: 123px; border-width: 1px; border-style: solid; border-color: rgb(171, 171, 171); box-sizing: border-box; height: 21.5px;"><br></td><td style="width: 123px; border-width: 1px; border-style: solid; border-color: rgb(171, 171, 171); box-sizing: border-box; height: 21.5px;"><br></td><td style="width: 123px; border-width: 1px; border-style: solid; border-color: rgb(171, 171, 171); box-sizing: border-box; height: 21.5px;"><br></td><td style="width: 123px; border-width: 1px; border-style: solid; border-color: rgb(171, 171, 171); box-sizing: border-box; height: 21.5px;"><br></td></tr><tr><td style="width: 123px; border-width: 1px; border-style: solid; border-color: rgb(171, 171, 171); box-sizing: border-box; height: 61.5px;" rowspan="2"><br><br></td><td style="width: 246px; border-width: 1px; border-style: solid; border-color: rgb(171, 171, 171); box-sizing: border-box; height: 40px;" colspan="2"><br><br></td><td style="width: 123px; border-width: 1px; border-style: solid; border-color: rgb(171, 171, 171); box-sizing: border-box; height: 40px;"><br></td></tr><tr><td style="width: 123px; border-width: 1px; border-style: solid; border-color: rgb(171, 171, 171); box-sizing: border-box; height: 21.5px;"><br></td><td style="width: 123px; border-width: 1px; border-style: solid; border-color: rgb(171, 171, 171); box-sizing: border-box; height: 21.5px;"><br></td><td style="width: 123px; border-width: 1px; border-style: solid; border-color: rgb(171, 171, 171); box-sizing: border-box; height: 21.5px;"><br></td></tr><tr><td style="width: 123px; border-width: 1px; border-style: solid; border-color: rgb(171, 171, 171); box-sizing: border-box; height: 21.5px;"><br></td><td style="width: 123px; border-width: 1px; border-style: solid; border-color: rgb(171, 171, 171); box-sizing: border-box; height: 21.5px;"><br></td><td style="width: 123px; border-width: 1px; border-style: solid; border-color: rgb(171, 171, 171); box-sizing: border-box; height: 21.5px;"><br></td><td style="width: 123px; border-width: 1px; border-style: solid; border-color: rgb(171, 171, 171); box-sizing: border-box; height: 21.5px;"><br></td></tr></tbody></table>';
```

Comment:
I removed more. I cannot remove border-color because I want to see the table cells when debugging if something goes wrong. Removing some font-* would change the position of each cell of the table, and we would manually calculate and update the table data in the test, so I kept them.